inc/object_item.class.php

```diff
@@ -82,13 +162,91 @@ static function showItemsForTarget(CommonDBTM $item) {
+            $elements[$itemL] = $object->getTypeName();
+         }
+         $rand = Dropdown::showFromArray('objectToAdd', $elements);
+         $paramsselsoft = ['objectToAdd' => '__VALUE__',
+                                'idMainobject' => $item->getID(),
+                                'mainobject' => $item->getType()];
+         Ajax::updateItemOnSelectEvent("dropdown_objectToAdd$rand", "show_".$rand,
+                                       $CFG_GLPI["root_doc"]."/plugins/genericobject/ajax/dropdownByItemtype.php",
+                                       $paramsselsoft);
+         echo "<span id='show_".$rand."'>&nbsp;</span>";
+         echo "</td><td width='20%'>";
+         echo "<input type='submit' name='add' value=\""._sx('button', 'Install')."\" class='submit'>";
+         echo "</td>";
+         echo "</tr>";
+         echo "</table>";
+         echo "</div>";
+         Html::closeForm();
+         echo "</div>";
+      }
+      echo "<div class='spaced'>";
+      //if ($canedit && $number) {
+         Html::openMassiveActionsForm('mass'.$item->getType().'_Item'.$rand);
+         $massiveactionparams = ['container' => 'mass'.$item->getType().'_Item'.$rand];
+         //Note : useless ?
+         $massiveactionparams['check_itemtype'] = $item->getType();
+         Html::showMassiveActions($massiveactionparams);
+      //}
+      echo "<table class='tab_cadre_fixehov'>";
+      $header_begin  = "<tr>";
+      $header_top    = '';
+      $header_bottom = '';
+      $header_end    = '';
+      //if ($canedit && $number) {
+         $header_top    .= "<th width='10'>".Html::getCheckAllAsCheckbox('mass'.$item->getType().'_Item'.$rand);
+         $header_top    .= "</th>";
+         $header_bottom .= "<th width='10'>".Html::getCheckAllAsCheckbox('mass'.$item->getType().'_Item'.$rand);
+         $header_bottom .= "</th>";
+      //}
+      $header_end .= "<th>".__('Type')."</th>";
+      $header_end .= "<th>".__('ID')."</th>";
+      $header_end .= "<th>".__('Name')."</th>";
+      echo $header_begin.$header_top.$header_end;
+      foreach ($item->getLinkedItemTypesAsArray() as $itemL) {
+         $object = new $itemL();
+         self::getItemListForObject($item->accesObjectType()->fields['itemtype'],
+            $object->accesObjectType()->fields['itemtype'], $item->fields['id']);
+      }
+      //if ($number) {
+         echo $header_begin.$header_bottom.$header_end;
+      //}
+      echo "</table>";
+      //if ($canedit && $number) {
+         $massiveactionparams['ontop'] = false;
+         Html::showMassiveActions($massiveactionparams);
+         Html::closeForm();
+      //}
+      return true;
+   }
+
    /**
     *
     * Enter description here ...
     * @since 2.2.0
     */
    static function registerType() {
-      Plugin::registerClass(get_called_class(), ['addtabon' => self::getLinkedItemTypes()]);
+      //Plugin::registerClass(get_called_class(), ['addtabon' => self::getLinkedItemTypes()]);
    }
```

Comment:
Don't comment code, change ```registerType ```function from object.class.php (line 237) to not call this function